composer.json

```diff
@@ -55,7 +56,8 @@
             "common\\http\\": "common/http",
             "entities\\": "entities",
             "services\\": "services",
-            "repositories\\": "repositories"
+            "repositories\\": "repositories",
+            "OpenEMR\\Encounter\\Services\\": "interface/patient_file/encounter/Services"
```

Comment:
https://github.com/openemr/openemr/pull/693#issuecomment-298178608 for my thoughts :)